### PR TITLE
Added run command

### PR DESCRIPTION
**Pull request by cpareek**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -18,5 +18,9 @@ locals {
   }), var.additional_script_path == null ? "" : file("${var.additional_script_path}")))
 
   additional_template_file = var.additional_script_uri != null ? format("%s%s%s", "[ ", "\"${var.additional_script_uri}\"", " ]") : "\"\""
+
+  #  Run command settings
+
```
Comment (bancey) [inline locals.tf]: Should there be some locals here? Looks like something is missing

**Review by TomKimber — DISMISSED** [review]

File: scripts/linux_cis.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+echo hello > /home/chirag_test.txt
```
Comment (bancey) [inline scripts/linux_cis.sh]: Is the linux hardening script to follow?

File: locals.tf (hunk 1 above)
Comment (cpareek) [reply locals.tf]: missed to remove that comment, had used locals to test something but then removed them :)

File: scripts/linux_cis.sh (hunk 2 above)
Comment (cpareek) [reply scripts/linux_cis.sh]: yupp at somepoint, just wanted to keep the PR smaller so  do all the Windows changes first and then Linux

**Review by bancey — APPROVED** [review]

**Pull request merged**